File: src/main/java/net/liyze/basin/core/commands/ForceStopCommand.java
package net.liyze.basin.core.commands;

import com.itranswarp.summer.context.annotation.Component;
import net.liyze.basin.core.Command;
import org.jetbrains.annotations.NotNull;

import java.util.List;

@Component
public class ForceStopCommand implements Command {
    @Override
    public void run(@NotNull List<String> args) {
        System.exit(0);
    }

    @SuppressWarnings("SpellCheckingInspection")
    @Override
    public @NotNull String Name() {
        return "forcestop";
    }
}

File: src/main/kotlin/net/liyze/basin/core/scan/ForceStopCommand.kt
package net.liyze.basin.core.scan

import net.liyze.basin.core.Command
import kotlin.system.exitProcess


class ForceStopCommand : Command {
    override fun run(args: List<String?>) {
        exitProcess(0)
    }

    @Suppress("SpellCheckingInspection")
    override fun Name(): String {
        return "forcestop"
    }
}
